# Spanish translation for gdebi
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the gdebi package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: gdebi\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2010-01-04 21:36+0100\n"
"PO-Revision-Date: 2006-05-11 21:02+0000\n"
"Last-Translator: Ricardo Pérez López <ricardo@iesdonana.org>\n"
"Language-Team: Spanish <es@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../gdebi:47
#, c-format
msgid ""
"usage: %prog [options] filename\n"
"For a graphical version run gdebi-gtk\n"
msgstr ""

#: ../gdebi:53 ../gdebi-gtk:53
msgid "Run non-interactive (dangerous!)"
msgstr "Ejecución no interactiva (¡peligroso!)"

#: ../gdebi:57
msgid "Set an APT configuration option"
msgstr ""

#: ../gdebi:61
msgid "Do not show progress information"
msgstr ""

#: ../gdebi:65
msgid "Simulate only and print a apt-get install compatible line to stderr"
msgstr ""

#: ../gdebi:67
msgid "Use alternative root dir"
msgstr ""

#: ../gdebi:75
#, fuzzy, c-format
msgid "gdebi error, file not found: %s\n"
msgstr "error de gdebi, archivo no encontrado: %s"

#: ../gdebi:94 ../gdebi:104
#, fuzzy
msgid "Need to be root to install packages"
msgstr "Falló la instalación del paquete '%s'"

#: ../gdebi:106 ../GDebi/GDebiCli.py:150
msgid "Do you want to install the software package? [y/N]:"
msgstr "¿Quiere instalar el paquete de software? [s/N]:"

#: ../gdebi-gtk:56
msgid "Auto close when the install is finished"
msgstr ""

#: ../gdebi-gtk:58
msgid "Use alternative datadir"
msgstr ""

#: ../gdebi-gtk:77
msgid "Software index is broken"
msgstr ""

#: ../gdebi-gtk:78
msgid ""
"This is a major failure of your software management system. Please check for "
"broken packages with synaptic, check the file permissions and correctness of "
"the file '/etc/apt/sources.list' and reload the software information with: "
"'sudo apt-get update' and 'sudo apt-get install -f'."
msgstr ""

#: ../data/gdebi.ui.h:1
msgid "  "
msgstr "  "

#: ../data/gdebi.ui.h:2
msgid "<b><big>   </big></b>"
msgstr "<b><big> </big></b>"

#: ../data/gdebi.ui.h:3 ../GDebi/GDebiKDE.py:138
msgid "<b>Maintainer:</b>"
msgstr "<b>Mantenedor:</b>"

#: ../data/gdebi.ui.h:4 ../GDebi/GDebiKDE.py:139
msgid "<b>Priority:</b>"
msgstr "<b>Prioridad:</b>"

#: ../data/gdebi.ui.h:5 ../GDebi/GDebiKDE.py:140
msgid "<b>Section:</b>"
msgstr "<b>Sección:</b>"

#: ../data/gdebi.ui.h:6 ../GDebi/GDebiKDE.py:141
msgid "<b>Size:</b>"
msgstr "<b>Tamaño:</b>"

#: ../data/gdebi.ui.h:7 ../GDebi/GDebiKDE.py:278
msgid "<b>To install the following changes are required:</b>"
msgstr "<b>Para instalar, se requieren los siguientes cambios</b>"

#: ../data/gdebi.ui.h:8 ../GDebi/GDebiKDE.py:137
msgid "<b>Version:</b>"
msgstr "<b>Versión:</b>"

#: ../data/gdebi.ui.h:9
msgid "Automatically close after the changes have been successfully applied"
msgstr ""

#: ../data/gdebi.ui.h:10 ../GDebi/GDebiKDE.py:132
msgid "Description"
msgstr "Descripción"

#: ../data/gdebi.ui.h:11
msgid "Description:"
msgstr "Descripción:"

#: ../data/gdebi.ui.h:12 ../GDebi/GDebiKDE.py:131 ../GDebi/GDebiKDE.py:133
#: ../GDebi/GDebiKDE.py:279
msgid "Details"
msgstr "Detalles"

#: ../data/gdebi.ui.h:13
msgid "GPL, see /usr/share/common-licenses/GPL"
msgstr "GPL, vea /usr/share/common-licenses/GPL"

#: ../data/gdebi.ui.h:14
#, fuzzy
msgid "Included files"
msgstr "Archivos incluidos"

#: ../data/gdebi.ui.h:15 ../data/gdebi.desktop.in.h:2
msgid "Install and view software packages"
msgstr "Instalar y ver paquetes de software"

#: ../data/gdebi.ui.h:16
msgid "Open..."
msgstr ""

#: ../data/gdebi.ui.h:17 ../data/gdebi.desktop.in.h:3
msgid "Package Installer"
msgstr "Instalador de paquetes"

#. first, we load all the default descriptions -- pyuic doesn't use
#. gettext as default (FIXME, copy code from language-selector)
#: ../data/gdebi.ui.h:18 ../GDebi/GDebiKDE.py:129
msgid "Package:"
msgstr "Paquete:"

#: ../data/gdebi.ui.h:19 ../GDebi/GDebiKDE.py:130
msgid "Status:"
msgstr "Estado:"

#: ../data/gdebi.ui.h:20
msgid "Terminal"
msgstr "Terminal"

#: ../data/gdebi.ui.h:21
msgid "_Details"
msgstr "_Detalles"

#: ../data/gdebi.ui.h:22
msgid "_File"
msgstr "_Archivo"

#: ../data/gdebi.ui.h:23
msgid "_Help"
msgstr "A_yuda"

#. img = gtk.Image()
#. img.set_from_stock(gtk.STOCK_APPLY,gtk.ICON_SIZE_BUTTON)
#. self.button_install.set_image(img)
#: ../data/gdebi.ui.h:24 ../GDebi/GDebi.py:246 ../GDebi/GDebi.py:286
msgid "_Install Package"
msgstr "_Instalar paquete"

#: ../data/gdebi.ui.h:25
msgid "_Refresh"
msgstr ""

#: ../data/gdebi.xml.in.h:1
msgid "Software package"
msgstr "Paquete de software"

#: ../data/gdebi.desktop.in.h:1
#, fuzzy
msgid "GDebi Package Installer"
msgstr "Instalador de paquetes"

#: ../GDebi/GDebi.py:77
msgid "Loading..."
msgstr "Cargando..."

#. set window title
#. set name
#: ../GDebi/GDebi.py:167 ../GDebi/GDebiKDE.py:179
#, python-format
msgid "Package Installer - %s"
msgstr "Instalador de paquetes - %s"

#: ../GDebi/GDebi.py:224
msgid "Package control data"
msgstr ""

#: ../GDebi/GDebi.py:227
msgid "Upstream data"
msgstr ""

#: ../GDebi/GDebi.py:233
msgid "Error reading filelist"
msgstr ""

#: ../GDebi/GDebi.py:243
msgid "Error: "
msgstr ""

#: ../GDebi/GDebi.py:258
msgid "Same version is already installed"
msgstr "Ya tiene instalada una version idéntica"

#: ../GDebi/GDebi.py:259
msgid "_Reinstall Package"
msgstr "_Reinstalar paquete"

#: ../GDebi/GDebi.py:302
msgid "Selection is a directory"
msgstr ""

#: ../GDebi/GDebi.py:307 ../GDebi/GDebi.py:312
#, python-format
msgid "Error reading file content '%s'"
msgstr ""

#: ../GDebi/GDebi.py:316
msgid "File content can not be extracted"
msgstr ""

#: ../GDebi/GDebi.py:327
#, python-format
msgid "<b>To be removed: %s</b>"
msgstr "<b>Se desinstalará: %s</b>"

#: ../GDebi/GDebi.py:329 ../GDebi/GDebiKDE.py:273
#, python-format
msgid "To be installed: %s"
msgstr "Se instalará: %s"

#: ../GDebi/GDebi.py:344
msgid "Open Software Package"
msgstr "Abrir paquete de software"

#: ../GDebi/GDebi.py:349
msgid "Software packages"
msgstr "Paquetes de software"

#: ../GDebi/GDebi.py:378
msgid "File not found"
msgstr ""

#: ../GDebi/GDebi.py:379
msgid "You tried to install a file that does not (or no longer) exist. "
msgstr ""

#. do it
#: ../GDebi/GDebi.py:389
msgid "Installing package file..."
msgstr "Instalando el archivo de paquete..."

#: ../GDebi/GDebi.py:391
msgid "Install unauthenticated software?"
msgstr "¿Instalar software sin autenticar?"

#: ../GDebi/GDebi.py:392
#, fuzzy
msgid ""
"Malicious software can damage your data and take control of your system.\n"
"\n"
"The packages below are not authenticated and could therefor be of malicious "
"nature."
msgstr ""
"El software malicioso podría dañar sus datos y tomar control de su sistema.\n"
"\n"
"Los siguientes paquetes no están autenticados y podrían ser de naturaleza "
"maliciosa."

#: ../GDebi/GDebi.py:419
msgid "You need to grant administrative rights to install software"
msgstr ""

#: ../GDebi/GDebi.py:420
#, fuzzy
msgid ""
"\n"
"It is a possible security risk to install packages files manually.\n"
"Install software from trustworthy software distributors only.\n"
msgstr ""
"<big><b>Necesita tener permisos administrativos para instalar software</b><"
"(big>\n"
"\n"
"Instalar archivos de paquetes manualmente constituye un posible riesgo de "
"seguridad. Instale sólo el software que provenga de distribuidores en los "
"que confíe."

#: ../GDebi/GDebi.py:434 ../GDebi/GDebi.py:482
msgid "Failed to install package file"
msgstr "Falló la instalación del paquete"

#: ../GDebi/GDebi.py:466 ../GDebi/GDebiKDE.py:325
msgid "Could not download all required files"
msgstr "No se han podido descargar todos los archivos requeridos"

#: ../GDebi/GDebi.py:467 ../GDebi/GDebiKDE.py:326
msgid "Please check your internet connection or installation medium."
msgstr ""
"Por favor, compruebe su conexión a Internet o el soporte de instalación."

#: ../GDebi/GDebi.py:471 ../GDebi/GDebiKDE.py:330
msgid "Could not install all dependencies"
msgstr "No pudieron instalarse todas las dependencias"

#: ../GDebi/GDebi.py:472 ../GDebi/GDebiKDE.py:331
msgid ""
"Usually this is related to an error of the software distributor. See the "
"terminal window for more details."
msgstr ""
"Esto normalmente es debido a un error del distribuidor del software. Vea la "
"ventana de terminal para más detalles."

#: ../GDebi/GDebi.py:487
#, fuzzy, python-format
msgid "Installing %s"
msgstr "Instalando '%s'..."

#. self.label_action.set_markup("<b><big>"+_("Package installed")+"</big></b>")
#. show the button
#. self.button_deb_install_close.set_sensitive(True)
#. self.button_deb_install_close.grab_default()
#: ../GDebi/GDebi.py:503 ../GDebi/GDebiKDE.py:350
#, fuzzy
msgid "Installation finished"
msgstr "Instalación completa"

#: ../GDebi/GDebi.py:505
#, python-format
msgid "Package '%s' was installed"
msgstr "Se ha instalado el paquete '%s'"

#: ../GDebi/GDebi.py:507 ../GDebi/GDebiKDE.py:354
#, python-format
msgid "Failed to install package '%s'"
msgstr "Falló la instalación del paquete '%s'"

#: ../GDebi/GDebi.py:509
msgid "Installation complete"
msgstr "Instalación completa"

#: ../GDebi/GDebi.py:516 ../GDebi/GDebiKDE.py:364
msgid "Failed to completely install all dependencies"
msgstr "Se produjo un fallo al instalar por completo todas las dependencias"

#: ../GDebi/GDebi.py:517 ../GDebi/GDebiKDE.py:365
msgid "To fix this run 'sudo apt-get install -f' in a terminal window."
msgstr ""
"Para arreglar esto, ejecute «sudo apt-get install -f» en la ventana de una "
"terminal."

#. ui
#: ../GDebi/GDebi.py:594 ../GDebi/KDEAptDialogs.py:72
#, python-format
msgid "Installing '%s'..."
msgstr "Instalando '%s'..."

#: ../GDebi/GDebi.py:679 ../GDebi/KDEAptDialogs.py:140
msgid "Installing dependencies..."
msgstr "Instalando dependencias..."

#. print "start()"
#: ../GDebi/GDebi.py:716 ../GDebi/KDEAptDialogs.py:201
#: ../GDebi/KDEAptDialogs.py:214 ../GDebi/KDEAptDialogs.py:216
msgid "Downloading additional package files..."
msgstr "Descargando archivos de paquetes adicionales..."

#: ../GDebi/GDebi.py:724 ../GDebi/KDEAptDialogs.py:214
#, fuzzy, python-format
msgid "File %s of %s at %sB/s"
msgstr "Archivo %s de %s a %s/s"

#: ../GDebi/GDebi.py:726 ../GDebi/KDEAptDialogs.py:216
#, fuzzy, python-format
msgid "File %s of %s"
msgstr "Archivo %s de %s a %s/s"

#. print "mediaChange %s %s" % (medium, drive)
#: ../GDebi/GDebi.py:733 ../GDebi/KDEAptDialogs.py:221
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Por favor, inserte '%s' en la unidad '%s'"

#: ../GDebi/DebPackage.py:139
#, python-format
msgid "Dependency is not satisfiable: %s\n"
msgstr "La dependencia no se puede satisfacer: %s\n"

#: ../GDebi/DebPackage.py:160
#, fuzzy, python-format
msgid "Conflicts with the installed package '%s'\n"
msgstr "Conflictos con el paquete instalado '%s'"

#. TRANSLATORS: the first '%s' is the package that breaks, the second the dependency that makes it break, the third the releation (e.g. >=) and the latest the version for the releation
#: ../GDebi/DebPackage.py:279
#, python-format
msgid "Breaks existing package '%s' dependency %s (%s %s)\n"
msgstr ""

#. TRANSLATORS: the first '%s' is the package that conflicts, the second the packagename that it conflicts with (so the name of the deb the user tries to install), the third is the relation (e.g. >=) and the last is the version for the relation
#: ../GDebi/DebPackage.py:291
#, python-format
msgid "Breaks exisiting package '%s' conflict: %s (%s %s)\n"
msgstr ""

#: ../GDebi/DebPackage.py:334
msgid "No Architecture field in the package"
msgstr ""

#: ../GDebi/DebPackage.py:339
#, python-format
msgid "Wrong architecture '%s'"
msgstr "Arquitectura incorrecta '%s'"

#. the deb is older than the installed
#: ../GDebi/DebPackage.py:345
msgid "A later version is already installed"
msgstr "Ya está instalada una versión posterior"

#. set progress information
#: ../GDebi/DebPackage.py:356
#, fuzzy
msgid "Analysing dependencies"
msgstr "Instalando dependencias..."

#: ../GDebi/DebPackage.py:374
msgid "Failed to satisfy all dependencies (broken cache)"
msgstr ""
"Se produjo un fallo al intentar satisfacer todas las dependencias (caché "
"dañado)"

#: ../GDebi/DebPackage.py:401
#, python-format
msgid "Cannot install '%s'"
msgstr "No se puede instalar '%s'"

#: ../GDebi/DebPackage.py:467
msgid ""
"Automatically decompressed:\n"
"\n"
msgstr ""

#: ../GDebi/DebPackage.py:473
msgid "Automatically converted to printable ascii:\n"
msgstr ""

#: ../GDebi/DebPackage.py:510
msgid "List of files could not be read, please report this as a bug"
msgstr ""

#: ../GDebi/DebPackage.py:513
#, python-format
msgid "IOError during filelist read: %s"
msgstr ""

#. Translators: it's for missing entries in the deb package,
#. e.g. a missing "Maintainer" field
#: ../GDebi/DebPackage.py:522
#, python-format
msgid "%s is not available"
msgstr "%s no está disponible"

#: ../GDebi/GDebiCli.py:58
msgid "Configuration items must be specified with a =<value>\n"
msgstr ""

#: ../GDebi/GDebiCli.py:64
#, python-format
msgid "Couldn't set APT option %s to %s\n"
msgstr ""

#: ../GDebi/GDebiCli.py:77
#, python-format
msgid "Unknown package type '%s', exiting\n"
msgstr ""

#: ../GDebi/GDebiCli.py:80
#, fuzzy
msgid "Failed to open the software package\n"
msgstr "Fallo al abrir el paquete de software"

#: ../GDebi/GDebiCli.py:81
#, fuzzy
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file.\n"
msgstr ""
"El paquete podría estar dañado, o puede que usted no tenga permisos para "
"abrir el archivo. Revise los permisos del archivo."

#: ../GDebi/GDebiCli.py:87
#, fuzzy
msgid "This package is uninstallable\n"
msgstr "Este paquete no es instalable"

#: ../GDebi/GDebiCli.py:96 ../GDebi/GDebiKDE.py:215
msgid "No description is available"
msgstr "No hay descripción disponible"

#: ../GDebi/GDebiCli.py:102
msgid "The following packages are UNAUTHENTICATED: "
msgstr "Los siguientes paquetes NO están AUTENTICADOS: "

#: ../GDebi/GDebiCli.py:106
msgid "Requires the REMOVAL of the following packages: "
msgstr "Requiere la DESINSTALACIÓN de los siguientes paquetes: "

#: ../GDebi/GDebiCli.py:111
msgid "Requires the installation of the following packages: "
msgstr "Requiere la instalación de los siguientes paquetes: "

#: ../GDebi/GDebiCli.py:125
#, fuzzy, python-format
msgid "Error during install: '%s'"
msgstr "Se instalará: %s"

#: ../GDebi/GDebiKDE.py:134
msgid "Included Files"
msgstr "Archivos incluidos"

#: ../GDebi/GDebiKDE.py:136 ../GDebi/GDebiKDE.py:261
#, fuzzy
msgid "&Install Package"
msgstr "_Instalar paquete"

#: ../GDebi/GDebiKDE.py:160
#, fuzzy
msgid "The package file does not exist"
msgstr "Este paquete no es instalable"

#: ../GDebi/GDebiKDE.py:161
msgid ""
"A nonexistent file has been selected for installation. Please select an "
"existing .deb package file."
msgstr ""

#. self.textLabel1_3_2.setText(_("Same version is already installed"))
#: ../GDebi/GDebiKDE.py:229
#, fuzzy
msgid "&Reinstall Package"
msgstr "_Reinstalar paquete"

#: ../GDebi/GDebiKDE.py:264
#, fuzzy
msgid "Re&install Package"
msgstr "_Reinstalar paquete"

#: ../GDebi/GDebiKDE.py:275
#, fuzzy, python-format
msgid "To be removed: %s"
msgstr "<b>Se desinstalará: %s</b>"

#: ../GDebi/GDebiKDE.py:352
msgid "<b>"
msgstr ""

#: ../GDebi/GDebiCommon.py:66
#, fuzzy
msgid "Broken dependencies"
msgstr "Instalando dependencias..."

#: ../GDebi/GDebiCommon.py:67
msgid ""
"Your system has broken dependencies. This application can not continue until "
"this is fixed. To fix it run 'gksudo synaptic' or 'sudo apt-get install -f' "
"in a terminal window."
msgstr ""

#: ../GDebi/GDebiCommon.py:84
#, python-format
msgid "'%s' is not a Debian package"
msgstr ""

#: ../GDebi/GDebiCommon.py:85
#, python-format
msgid ""
"The MIME type of this file is '%s' and can not be installed on this system."
msgstr ""

#: ../GDebi/GDebiCommon.py:89
#, python-format
msgid "Could not open '%s'"
msgstr "No se pudo abrir '%s'"

#: ../GDebi/GDebiCommon.py:90
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file."
msgstr ""
"El paquete podría estar dañado, o puede que usted no tenga permisos para "
"abrir el archivo. Revise los permisos del archivo."

#: ../GDebi/GDebiCommon.py:106
msgid "Same version is available in a software channel"
msgstr "En un canal de software está disponible la misma versión"

#: ../GDebi/GDebiCommon.py:107
msgid "You are recommended to install the software from the channel instead."
msgstr "Se recomienda que instale el software desde el canal."

#: ../GDebi/GDebiCommon.py:111
msgid "An older version is available in a software channel"
msgstr "En un canal de software hay disponible una versión anterior"

#: ../GDebi/GDebiCommon.py:112
msgid ""
"Generally you are recommended to install the version from the software "
"channel, since it is usually better supported."
msgstr ""
"En general, se recomienda que instale software desde los canales de "
"software, ya que normalmente tiene mejor soporte."

#: ../GDebi/GDebiCommon.py:117
msgid "A later version is available in a software channel"
msgstr "En un canal de software hay disponible una versión posterior"

#: ../GDebi/GDebiCommon.py:119
msgid ""
"You are strongly advised to install the version from the software channel, "
"since it is usually better supported."
msgstr ""
"Se recomienda encarecidamente que instale esa versión desde el canal de "
"software, ya que normalmente tiene mejor soporte."

#: ../GDebi/GDebiCommon.py:127
msgid "All dependencies are satisfied"
msgstr "Se satisfacen todas las dependencias"

#. FIXME: use ngettext here
#: ../GDebi/GDebiCommon.py:130
#, python-format
msgid "Requires the <b>removal</b> of %s packages\n"
msgstr "Requiere la <b>desinstalación</b> de %s paquetes\n"

#: ../GDebi/GDebiCommon.py:132
#, python-format
msgid "Requires the installation of %s packages"
msgstr "Requiere la instalación de %s paquetes"

#: ../GDebi/GDebiCommon.py:140 ../GDebi/GDebiCommon.py:158
msgid "Only one software management tool is allowed to run at the same time"
msgstr ""
"Sólo se permite la ejecución simultánea de una única herramienta de gestión "
"de software"

#: ../GDebi/GDebiCommon.py:142 ../GDebi/GDebiCommon.py:160
msgid ""
"Please close the other application e.g. 'Update Manager', 'aptitude' or "
"'Synaptic' first."
msgstr ""
"Por favor, cierre cualquier otra aplicación como «Gestor de actualizaciones», "
"«aptitude» o «Synaptic» primero."

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../GDebi/KDEAptDialogs.py:223
msgid "Media Change"
msgstr ""

#~ msgid "gdebi"
#~ msgstr "gdebi"

#~ msgid "translator-credits"
#~ msgstr "Daniel Fuertes Pérez, Francisco Villalobos Palma"

#~ msgid "(c) 2005-2006 Canonical"
#~ msgstr "(c) 2005-2006 Canonical"

#~ msgid "Installing package file"
#~ msgstr "Instalando archivo de paquete"

#~ msgid "Usage: %s [package.deb]"
#~ msgstr "Uso: %s [paquete.deb]"

#~ msgid ""
#~ "To use the graphical user interface, right-click\n"
#~ "on a '.deb' software package in the file browser\n"
#~ "and select: Open with 'GDebi Package Installer'."
#~ msgstr ""
#~ "Para usar la interfaz grafica, pulse con el botón derecho\n"
#~ "sobre un paquete '.deb' en el explorador de archivos\n"
#~ "y seleccione: Abrir con 'Instalador de paquetes GDebi'."

#~ msgid "_Grant"
#~ msgstr "_Permitir"
